packages/wrangler/src/r2/lifecycle.ts

```diff
@@ -0,0 +1,412 @@
+					`Enter the number of days after which to ${formatActionDescription(action)}`
+				);
+			}
+			if (!isNonNegativeNumber(String(conditionValue))) {
+				throw new UserError("Must be a positive number.");
+			}
+
+			conditionType = "Age";
+			conditionValue = Number(conditionValue) * 86400; // Convert days to seconds
+
+			newRule.abortMultipartUploadsTransition = {
+				condition: {
+					maxAge: conditionValue,
+					type: conditionType,
+				},
+			};
+		} else {
+			if (expireDays !== undefined) {
+				conditionType = "Age";
+				conditionValue = expireDays;
+			} else if (iaTransitionDays !== undefined) {
+				conditionType = "Age";
+				conditionValue = iaTransitionDays;
+			} else if (expireDate !== undefined) {
+				conditionType = "Date";
+				conditionValue = expireDate;
+			} else if (iaTransitionDate !== undefined) {
+				conditionType = "Date";
+				conditionValue = iaTransitionDate;
+			} else {
+				conditionValue = await prompt(
+					`Enter the number of days or a date (YYYY-MM-DD) after which to ${formatActionDescription(action)}`
+				);
+				if (
+					!isNonNegativeNumber(String(conditionValue)) &&
+					!isValidDate(String(conditionValue))
+				) {
+					throw new UserError(
+						"Must be a positive number or a valid date in the YYYY-MM-DD format."
+					);
+				}
+			}
+
+			if (isNonNegativeNumber(String(conditionValue))) {
+				conditionType = "Age";
+				conditionValue = Number(conditionValue) * 86400; // Convert days to seconds
+			} else if (isValidDate(String(conditionValue))) {
+				conditionType = "Date";
+				const date = new Date(`${conditionValue}T00:00:00.000Z`);
+				conditionValue = date.toISOString();
+			} else {
+				throw new UserError("Invalid condition input.");
+			}
+
+			if (action === "expire") {
+				newRule.deleteObjectsTransition = {
+					condition: {
+						[conditionType === "Age" ? "maxAge" : "date"]: conditionValue,
+						type: conditionType,
+					},
+				};
+			} else if (action === "transition") {
+				newRule.storageClassTransitions = [
+					{
+						condition: {
+							[conditionType === "Age" ? "maxAge" : "date"]: conditionValue,
+							type: conditionType,
+						},
+						storageClass: "InfrequentAccess",
+					},
+				];
+			}
+		}
+	}
+
+	lifecycleRules.push(newRule);
+	logger.log(`Adding lifecycle rule '${id}' to bucket '${bucket}'...`);
+	await putLifecycleRules(accountId, bucket, lifecycleRules, jurisdiction);
+	logger.log(`✨ Added lifecycle rule '${id}' to bucket '${bucket}'.`);
+}
+
+export function RemoveOptions(yargs: CommonYargsArgv) {
+	return yargs
+		.positional("bucket", {
+			describe: "The name of the R2 bucket to remove a lifecycle rule from",
+			type: "string",
+			demandOption: true,
+		})
+		.option("id", {
+			describe: "The unique identifier of the lifecycle rule to remove",
+			type: "string",
+			demandOption: true,
+			requiresArg: true,
+		})
+		.option("jurisdiction", {
+			describe: "The jurisdiction where the bucket exists",
```

Comment:
nit
```suggestion
			describe: "The jurisdiction where the R2 bucket exists",
```

Reply:
This is consistent with other jurisdiction option descriptions